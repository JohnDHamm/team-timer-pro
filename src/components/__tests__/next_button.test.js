import React from 'react'
import renderer from 'react-test-renderer'
// import NextButton from '../next_button'

describe('<NextButton />', () => {
//   const label = 'button label';

  // test('it renders correctly as enabled', () => {
  //   const tree = renderer.create(<NextButton disabled={false} label={label} />).toJSON();
  //   expect(tree).toMatchSnapshot();
  // });

  // test('it renders correctly as disabled', () => {
  //   const tree = renderer.create(<NextButton disabled={true} label={label} />).toJSON();
  //   expect(tree).toMatchSnapshot();
  // });

  test('test placeholder', () => {
    expect(true).toBe(true)
  })
});
