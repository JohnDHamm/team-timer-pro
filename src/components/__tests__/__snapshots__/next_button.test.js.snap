// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NextButton /> it renders correctly as disabled 1`] = `
<View>
  <View
    style={
      Object {
        "alignItems": "center",
        "flexDirection": "row",
        "justifyContent": "center",
      }
    }
  >
    <Text
      style={
        Array [
          Object {
            "color": "#6E5EAB",
            "fontFamily": "dosis-medium",
            "fontSize": 20,
            "paddingBottom": 5,
          },
          Object {
            "color": "#CCCCCC",
          },
        ]
      }
    >
      button label
    </Text>
    <Image
      source={1}
      style={
        Array [
          Object {
            "height": 25.3324889170361,
            "width": 40,
          },
          Object {
            "tintColor": "#CCCCCC",
          },
        ]
      }
    />
  </View>
</View>
`;

exports[`<NextButton /> it renders correctly as enabled 1`] = `
<View>
  <View
    accessible={true}
    isTVSelectable={true}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "alignItems": "center",
        "flexDirection": "row",
        "justifyContent": "center",
        "opacity": 1,
      }
    }
  >
    <Text
      style={
        Object {
          "color": "#6E5EAB",
          "fontFamily": "dosis-medium",
          "fontSize": 20,
          "paddingBottom": 5,
        }
      }
    >
      button label
    </Text>
    <Image
      source={1}
      style={
        Object {
          "height": 25.3324889170361,
          "width": 40,
        }
      }
    />
  </View>
</View>
`;
